Sources/ArcGISToolkit/Components/Form/FormInputs/SingleLineTextEntry.swift

```diff
@@ -51,8 +51,10 @@ struct SingleLineTextEntry: View {
         HStack {
             TextField(element.label, text: $text, prompt: Text(element.hint ?? "").foregroundColor(.secondary))
                 .focused($isFocused)
-            if isFocused && !text.isEmpty {
+                .accessibilityIdentifier("\(element.label) Text Field")
```

Comment:
These have been added for testing purposes only.

Reply:
@mhdostal You're thinking of [`accessibilityLabel`](https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel), which is localized.